### ISSUE TITLE
PDAL error : class pdal::LasHeader' has no member named 'incompatibleSrs

### ISSUE THREAD

**Issue by kikislater** [bartoszek/AUR-cloudcompare#7]
Hi,

It doesn't build after latest release of pdal 

```
$ makepkg -fsi
==> Making package: cloudcompare 2.11.3-2 (lun. 26 juil. 2021 09:36:16)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating cloudcompare git repo...
Fetching origin
  -> Updating cloudcompare-cork git repo...
Fetching origin
  -> Found constexpr.patch
  -> Found cork.patch
  -> Found pcl.patch
  -> Found CloudCompare.desktop
  -> Found ccViewer.desktop
==> Validating source files with sha256sums...
    cloudcompare ... Skipped
    cloudcompare-cork ... Skipped
    constexpr.patch ... Passed
    cork.patch ... Passed
    pcl.patch ... Passed
    CloudCompare.desktop ... Passed
    ccViewer.desktop ... Passed
==> Extracting sources...
  -> Creating working copy of cloudcompare git repo...
Reset branch 'makepkg'
  -> Creating working copy of cloudcompare-cork git repo...
Switched to a new branch 'makepkg'
==> Starting prepare()...
Checking patch CC/include/CCConst.h...
Checking patch libs/qCC_db/ccCameraSensor.cpp...
Checking patch libs/qCC_db/ccColorTypes.h...
Applied patch CC/include/CCConst.h cleanly.
Applied patch libs/qCC_db/ccCameraSensor.cpp cleanly.
Applied patch libs/qCC_db/ccColorTypes.h cleanly.
Checking patch plugins/core/Standard/qPCL/PclUtils/filters/MLSSmoothingUpsampling.cpp...
Checking patch plugins/core/Standard/qPCL/PclUtils/utils/my_point_types.h...
Applied patch plugins/core/Standard/qPCL/PclUtils/filters/MLSSmoothingUpsampling.cpp cleanly.
Applied patch plugins/core/Standard/qPCL/PclUtils/utils/my_point_types.h cleanly.
Checking patch Makefile...
Checking patch src/mesh/mesh.isct.tpp...
Checking patch src/util/prelude.h...
Applied patch Makefile cleanly.
Applied patch src/mesh/mesh.isct.tpp cleanly.
Applied patch src/util/prelude.h cleanly.
==> Removing existing $pkgdir/ directory...
==> Starting build()...
  -> Build Cork lib
make: Entering directory '/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare-cork'
Compiling obj/cork.o
Compiling obj/util/timer.o
Compiling obj/util/log.o
Compiling obj/isct/empty3d.o
Compiling obj/file_formats/files.o
Compiling obj/file_formats/ifs.o
Compiling obj/file_formats/off.o
Compiling obj/main.o
Bundling lib/libcork.a
Linking cork command line tool
make: Leaving directory '/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare-cork'
  -> Build CloudCompare
-- __cplusplus is 201103
--   --> Do not link with Boost.Thread
-- Found Boost: /usr/lib64/cmake/Boost-1.76.0/BoostConfig.cmake (found suitable version "1.76.0", minimum required is "1.48")  
-- Boost include dirs: /usr/include
-- Boost libraries:    
-- Targetting Ninja
-- Using /usr/bin/c++ compiler.
-- Found Boost: /usr/lib64/cmake/Boost-1.76.0/BoostConfig.cmake (found suitable version "1.76.0", minimum required is "1.48") found components: thread system 
-- Build type: Release
-- USING CXXFLAGS = '-march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fpermissive -DSUPPORT_TOPO_STREAM_OPERATORS -Wno-deprecated-declarations -std=c++11 -fPIC -std=c++11 -fPIC -O3 -DNDEBUG'
-- USING EXEFLAGS = '-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now '
-- Requested component: Core
-- Requested component: MPFR
-- Requested component: GMP
-- OpenMP found
-- OpenMP found
-- Eigen found (include: /usr/include/eigen3, version: 3.3.9)
-- Found Boost: /usr/include (found suitable version "1.76.0", minimum required is "1.55.0") found components: system filesystem date_time iostreams serialization regex 
** WARNING ** io features related to pcap will be disabled
** WARNING ** io features related to png will be disabled
** WARNING ** io features related to libusb-1.0 will be disabled
-- QHULL found (include: /usr/include, lib: optimized;/usr/lib/libqhull.so;debug;/usr/lib/libqhull.so)
-- looking for PCL_COMMON
-- looking for PCL_KDTREE
-- looking for PCL_OCTREE
-- looking for PCL_SEARCH
-- looking for PCL_SAMPLE_CONSENSUS
-- looking for PCL_FILTERS
-- looking for PCL_2D
-- looking for PCL_GEOMETRY
-- looking for PCL_IO
-- looking for PCL_FEATURES
-- looking for PCL_ML
-- looking for PCL_SEGMENTATION
-- looking for PCL_VISUALIZATION
-- looking for PCL_SURFACE
-- looking for PCL_REGISTRATION
-- looking for PCL_KEYPOINTS
-- looking for PCL_TRACKING
-- looking for PCL_RECOGNITION
-- looking for PCL_STEREO
-- looking for PCL_OUTOFCORE
-- looking for PCL_PEOPLE
-- Auto-add 3rd party plugin: /home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/plugins/3rdParty/qRDBIO
-- Configuring done
-- Generating done
-- Build files have been written to: /home/sylvain/.cache/yay/cloudcompare/src/build
ninja: Entering directory `build'
[6/629] Generating CloudCompare_fr.qm
Updating '/home/sylvain/.cache/yay/cloudcompare/src/build/qCC/translations/CloudCompare_fr.qm'...
    Generated 509 translation(s) (386 finished and 123 unfinished)
    Ignored 2447 untranslated source text(s)
[9/629] Generating CloudCompare_ja.qm
Updating '/home/sylvain/.cache/yay/cloudcompare/src/build/qCC/translations/CloudCompare_ja.qm'...
    Generated 663 translation(s) (534 finished and 129 unfinished)
    Ignored 2293 untranslated source text(s)
[52/629] Generating CloudCompare_pt.qm
Updating '/home/sylvain/.cache/yay/cloudcompare/src/build/qCC/translations/CloudCompare_pt.qm'...
    Generated 573 translation(s) (97 finished and 476 unfinished)
    Ignored 2383 untranslated source text(s)
[53/629] Generating CloudCompare_ru.qm
Updating '/home/sylvain/.cache/yay/cloudcompare/src/build/qCC/translations/CloudCompare_ru.qm'...
    Generated 2956 translation(s) (2956 finished and 0 unfinished)
[129/629] Automatic MOC for target QCC_GL_LIB
AutoMoc: /home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/libs/qCC_glWindow/ccGLWidget.h:0: Note: No relevant classes found. No output generated.
[202/629] Building CXX object plugins/...PDAL_IO_PLUGIN.dir/src/LASFilter.cpp.o
FAILED: plugins/core/IO/qPDALIO/CMakeFiles/QPDAL_IO_PLUGIN.dir/src/LASFilter.cpp.o 
/usr/bin/c++ -DBOOST_ALL_NO_LIB -DBOOST_SYSTEM_DYN_LINK -DBOOST_THREAD_DYN_LINK -DCC_DXF_SUPPORT -DCC_GDAL_SUPPORT -DCC_SHP_SUPPORT -DCGAL_USE_CORE=1 -DQPDAL_IO_PLUGIN_EXPORTS -DQT_CONCURRENT_LIB -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NO_DEBUG -DQT_OPENGLEXTENSIONS_LIB -DQT_OPENGL_LIB -DQT_USE_QSTRINGBUILDER -DQT_WIDGETS_LIB -DSCALAR_TYPE_FLOAT -I/home/sylvain/.cache/yay/cloudcompare/src/build/plugins/core/IO/qPDALIO -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/plugins/core/IO/qPDALIO -I/home/sylvain/.cache/yay/cloudcompare/src/build/plugins/core/IO/qPDALIO/QPDAL_IO_PLUGIN_autogen/include -I/usr/include/qt/QtPrintSupport -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/contrib/dxflib-3.17.0/src -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/contrib/shapelib -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/plugins/core/IO/qPDALIO/src -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/plugins -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/qCC/../common -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/CC/include -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/libs/CCFbo/include -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/libs/qCC_io -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/libs/qCC_db -I/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/libs/qCC_glWindow -isystem /usr/include/qt -isystem /usr/include/qt/QtOpenGL -isystem /usr/include/qt/QtWidgets -isystem /usr/include/qt/QtGui -isystem /usr/include/qt/QtCore -isystem /usr/lib/qt/mkspecs/linux-g++ -isystem /usr/include/qt/QtConcurrent -isystem /usr/include/qt/QtOpenGLExtensions -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fpermissive -DSUPPORT_TOPO_STREAM_OPERATORS -Wno-deprecated-declarations -std=c++11 -fPIC -fopenmp -DUSE_TBB -O3 -DNDEBUG -fPIC -fPIC -frounding-math -std=gnu++14 -MD -MT plugins/core/IO/qPDALIO/CMakeFiles/QPDAL_IO_PLUGIN.dir/src/LASFilter.cpp.o -MF plugins/core/IO/qPDALIO/CMakeFiles/QPDAL_IO_PLUGIN.dir/src/LASFilter.cpp.o.d -o plugins/core/IO/qPDALIO/CMakeFiles/QPDAL_IO_PLUGIN.dir/src/LASFilter.cpp.o -c /home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/plugins/core/IO/qPDALIO/src/LASFilter.cpp
/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/plugins/core/IO/qPDALIO/src/LASFilter.cpp: In lambda function:
/home/sylvain/.cache/yay/cloudcompare/src/cloudcompare/plugins/core/IO/qPDALIO/src/LASFilter.cpp:1360:52: error: 'class pdal::LasHeader' has no member named 'incompatibleSrs'
 1360 |                                 else if (lasHeader.incompatibleSrs())
      |                                                    ^~~~~~~~~~~~~~~
[210/629] Building CXX object plugins/...QE57_IO_PLUGIN.dir/src/E57Filter.cpp.o
ninja: build stopped: subcommand failed.
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by kikislater** [comment]
Ok ... there is a fix here => 
https://github.com/CloudCompare/CloudCompare/commit/f3038dcdeb0491c4a653c2ee6fb017326eb676a3

**Comment by bartoszek** [comment]
Fixed cc9d6156ed26439c85ba99c0f78cc3b53f051627
